protoc-gen-java-snowpea/README.md

```diff
@@ -0,0 +1,83 @@
+# Protoc Plugin: protoc-gen-java-snowpea
+
+This plugin generates tests for Google Event Types Library for Java to ensure the ability to unmarshal data.
+
+## Things to Know
+
+* This plugin generates a test file per "main" event type
+    * The "main" event type is determined by the protobuf's message "Data" suffix, i.e. generate MessagePublishedData and not PubSubMessage
```

Comment:
This seems like a reasonable assumption but I expect is not a standard. I'm somewhat concerned a new event source could be created where the word data just happens to be used in a data structure type because it's not codified as a reserved type name suffix.

In Go, I drove the code generation from the events.proto. Messages in the events.proto file map to the event types, and refer to the top-level data structure. You can see how I processed that in Go here: https://github.com/googleapis/google-cloudevents-go/blob/906043c383dff7a853ce7c4e8da2662c1361d919/generators/protoc-gen-go-googlecetypes/generator.go#L114

I found on discussing with @marcgovi that the presence and location of the events.proto is also not a standard, but making it a more explicit rule is being explored.

It's unclear to me if the "Data" suffix rule is safe in practice, but I'm comfortable moving this PR forward and opening an issue to follow-up.

Reply:
Created issue to track this: https://github.com/googleapis/google-cloudevents-java/issues/128